<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VpMain</class>
 <widget class="QMainWindow" name="VpMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>950</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>VP Plots</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>10</y>
      <width>161</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ControlArea"/>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1161</width>
      <height>871</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1160</width>
        <height>870</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="FormLayoutType_01"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1160</width>
        <height>870</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="FormLayoutType_02"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1161</width>
        <height>871</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="FormLayoutType_03"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="formLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1161</width>
        <height>871</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="FormLayoutType_04"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QWidget" name="formLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1161</width>
        <height>871</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="FormLayoutType_05"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QWidget" name="formLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1161</width>
        <height>871</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="FormLayoutType_06"/>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="ControlFrame">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>20</y>
      <width>139</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="RB_Type_01">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>VP attributes</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RB_Type_02">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>VP histograms</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_Next">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_Prev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Prev</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RB_Type_04">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vib activity (all)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RB_Type_05">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vib activity (each)</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="DateEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="RB_Type_03">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>VP error bars</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RB_Type_06">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Node attributes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>250</y>
      <width>161</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="StatusHeaderLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
       <widget class="QLabel" name="StatusLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>141</width>
          <height>211</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
       <widget class="QLabel" name="StatusDatabaseLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Database</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="StatusDestinationLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>890</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Destination folder</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ActionDefaultDatabase"/>
    <addaction name="ActionSelectDatabase"/>
    <addaction name="separator"/>
    <addaction name="ActionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSave_plots">
    <property name="title">
     <string>Save plots</string>
    </property>
    <addaction name="ActionDestinationFolder"/>
    <addaction name="separator"/>
    <addaction name="ActionSaveAll"/>
    <addaction name="ActionSaveType_01"/>
    <addaction name="ActionSaveType_02"/>
    <addaction name="ActionSaveType_03"/>
    <addaction name="ActionSaveType_04"/>
    <addaction name="ActionSaveType_05"/>
    <addaction name="ActionSaveType_06"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSave_plots"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ActionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="ActionSelectDatabase">
   <property name="text">
    <string>Select database</string>
   </property>
  </action>
  <action name="ActionDefaultDatabase">
   <property name="text">
    <string>Default database</string>
   </property>
  </action>
  <action name="ActionSaveAll">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="ActionSaveType_01">
   <property name="text">
    <string>VP attributes</string>
   </property>
  </action>
  <action name="ActionSaveType_02">
   <property name="text">
    <string>VP histograms</string>
   </property>
  </action>
  <action name="ActionSaveType_03">
   <property name="text">
    <string>VP error bars</string>
   </property>
  </action>
  <action name="ActionSaveType_04">
   <property name="text">
    <string>Vib activity (all)</string>
   </property>
  </action>
  <action name="ActionSaveType_05">
   <property name="text">
    <string>Vib activity (each)</string>
   </property>
  </action>
  <action name="ActionDestinationFolder">
   <property name="text">
    <string>Destination folder</string>
   </property>
  </action>
  <action name="ActionSaveType_06">
   <property name="text">
    <string>Node attributes</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>RB_Type_01</tabstop>
  <tabstop>RB_Type_02</tabstop>
  <tabstop>RB_Type_03</tabstop>
  <tabstop>RB_Type_04</tabstop>
  <tabstop>RB_Type_05</tabstop>
  <tabstop>RB_Type_06</tabstop>
  <tabstop>PB_Prev</tabstop>
  <tabstop>PB_Next</tabstop>
  <tabstop>DateEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
